src/main/java/org/dspace/ctask/replicate/AbstractPackagerTask.java

```diff
@@ -65,20 +69,26 @@ protected PackageParameters loadPackagerParameters(String moduleName)
     {
         //Load up the replicate-mets.cfg file & all settings inside it
         List<String> moduleProps = configurationService.getPropertyKeys(moduleName);
-        
+
         PackageParameters pkgParams = new PackageParameters();
-        
+
         //If our config file doesn't load properly, we'll return null
         if(moduleProps!=null)
-        {    
+        {
             //loop through all properties in the config file
             for(String property : moduleProps)
             {
+                //Set propertyName, removing leading module name (if applicable)
+                String propertyName = property;
+                if(propertyName.startsWith(moduleName + ".")) {
+                    propertyName = property.replaceFirst(moduleName + ".", "");
```

Comment:
@bbranan : while I'm sure this code works, it looks a bit odd (and potentially fragile).  It's weird to check for the `moduleName` in `propertyName` but then replace it in `property`.  In other words, this likely only works cause you set them equal to each other. If that ever changes, this code would break.  Probably best to use the same variable for both the `startsWith` and the `replaceFirst`.

Long story short, might want to change this to ` propertyName = propertyName.replaceFirst(moduleName + ".", "");`

Reply:
👍